inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java

```diff
@@ -70,12 +78,21 @@ public class AuditServiceImpl implements AuditService {
     private static final String HOUR_FORMAT = "yyyy-MM-dd HH";
     private static final String DAY_FORMAT = "yyyy-MM-dd";
 
+    @Value("#{'${audit.admin.ids:3,4,5,6,7,8}'.split(',')}")
```

Comment:
1. The number is not clear for users.
2. How to set those ids? In a properties file?

Reply:
1. doc and inlong audit definition url link added.
2. set those ids in application.properties like:
audit.admin.ids = 3,4,5,6,7,8
audit.user.ids = 3,8